lib/rules/max-lines-per-function.js

```diff
@@ -190,11 +193,24 @@ module.exports = {
 
             if (lineCount > maxLines) {
                 const name = upperCaseFirst(astUtils.getFunctionNameWithKind(funcNode));
+                const linesExceed = lineCount - maxLines;
+                const srcCodeLinesLength = [...sourceCode.lines].pop().length;
+
+                const loc = {
+                    start: {
+                        line: lines[maxLines].lineNumber,
+                        column: 0
+                    },
```

Comment:
This would always be the same line in the file. For example, if the max is 5, it will report line 6.

![image](https://user-images.githubusercontent.com/44349756/139720913-462459ff-4bf2-4b8c-aea0-49fba4d0f956.png)

We should find the first line that goes above the max in the function (line 13 in this screenshot).